src/RepoAuditor/Plugins/GitHub/StandardQueryRequirements/Impl/StandardEnableRequirementImpl.py

```diff
@@ -28,13 +27,13 @@ def __init__(
         rationale: str,
         subject: Optional[str] = None,
         *,
+        default_value: bool,

```

Comment:
Same comment about parameter reordering. I'd prefer that we suppress the \`ruff\` error.